dev/tools/localization/gen_l10n.dart

```diff
@@ -234,7 +238,7 @@ String genPluralMethod(Map<String, dynamic> bundle, String key) {
     ...genIntlMethodArgs(bundle, key),
   ];
 
-  for(String pluralKey in pluralIds.keys) {
+  for (String pluralKey in pluralIds.keys.toList()..sort()) {
```

Comment:
I don't think we need to sort pluralIds (which should be const instead of final). It's a LinkedHashMap so the order of the keys is stable and matches the order they were defined in.